pom.xml

```diff
@@ -246,17 +246,25 @@
 		<dependency>
 			<groupId>org.apache.tomcat.embed</groupId>
 			<artifactId>tomcat-embed-core</artifactId>
-			<version>9.0.71</version>
+			<version>9.0.95</version>

```

Comment:
This was brought to attention here : https://github.com/kawansoft/aceql-http/pull/30